srv/salt/_modules/cephprocesses.py

```diff
@@ -27,7 +27,7 @@
              'mgr': ['ceph-mgr'],
              'storage': ['ceph-osd'],
              'mds': ['ceph-mds'],
-             'igw': [],
```

Comment:
Is this needed?  lrbd will never get checked below and gives the impression that the service needs to be up.

Reply:
You get the res['up'/'down'] map based on the services from `processes`.